pkg/tfgen/examples_coverage_exporter.go

```diff
@@ -440,22 +440,20 @@ func (ce *coverageExportUtil) exportMarkdown(outputDirectory string, fileName st
 	return os.WriteFile(targetFile, []byte(out), 0600)
 }
 
-// The fourth mode, which simply gives the provider name, and success percentage.
-func (ce *coverageExportUtil) exportHumanReadable(outputDirectory string, fileName string) error {
-
-	// The Coverage Tracker data structure is flattened to gather statistics about each language
-	type LanguageStatistic struct {
-		Total     int
-		Successes int
-	}
+// The Coverage Tracker data structure is flattened to gather statistics about each language
+type LanguageStatistic struct {
```

Comment:
Beware when you lift structs to top-level with names starting from Capital Letter they get exported/public.

If you do not need it public call it "languageStatistic".